engine/execution/computation/manager_benchmark_test.go

```diff
@@ -128,6 +128,7 @@ func BenchmarkComputeBlock(b *testing.B) {
 
 	me := new(module.Local)
 	me.On("NodeID").Return(flow.ZeroID)
+	me.On("Sign", mock.Anything, mock.Anything).Return(nil, nil)
```

Comment:
NOTE: I've sneak in two fixes to the benchmark.  Sign was broken when I refactor receipt generation into computation result, and the ledger update was broken when I refactor to computer to be idempotent.